mitiq/tests/test_factories.py

```diff
@@ -23,43 +23,48 @@ def f_non_lin(x: float) -> float:
     return A + B * x + C * x ** 2
 
 
-def apply_algorithm(algo_class: Factory, f: Callable[[float], float], order: float = None) -> float:
+def apply_algorithm(algo_object: Factory, f: Callable[[float], float]) -> float:
```

Comment:
The fact that this is used to simplify tests suggests that something similar should be in the main library to simplify usage.

Reply:
Good point! And indeed, as you suggested, we now have a function for this: `zne.run_factory`.